aiortc/mediastreams.py

```diff
@@ -7,6 +7,8 @@
 from pyee import EventEmitter
 
 AUDIO_PTIME = 0.020  # 20ms audio packetization
+AUDIO_CLOCK_RATE = 8000
```

Comment:
This kind of bothers me, these are not actually constants, they depend very much on what is negotiated.

Video is different, the clockrate is truly a constant. (VIDEO_PTIME is not..)

Reply:
This constans used only for AudioStreamTrack, that returns dummy frames. As a suggestion we can rename this const to sothing like `DUMMY_AUDIO_CLOK_RATE`